qiskit/circuit/library/standard_gates/xx_plus_yy.py

```diff
@@ -42,7 +42,7 @@ class XXPlusYYGate(Gate):
         \newcommand{\th}{\frac{\theta}{2}}
 
         R_{XX+YY}(\theta, \beta)\ q_0, q_1 =
-          RZ_1(\beta) \cdot \exp\left(-i \frac{\theta}{2} \frac{XX+YY}{2}\right) \cdot RZ_1(-\beta) =
+          RZ_1(-\beta) \cdot \exp\left(i \frac{\theta}{2} \frac{XX+YY}{2}\right) \cdot RZ_1(\beta) =
             \begin{pmatrix}
                 1 & 0                     & 0                    & 0  \\
                 0 & \cos\left(\th\right)             & i\sin\left(\th\right)e^{i\beta} & 0  \\
```

Comment:
Regardless of the result of the discussion about how the gate _should_ be defined, I did this maths myself and agree with the PR's version matrix expressions.